src/apb/dat/Makefile.j2

```diff
@@ -0,0 +1,21 @@
+DOCKERHOST = {{ apb_dict['dockerhost'] }}
+DOCKERORG = {{ apb_dict['app_org'] }}
+IMAGENAME = {{ apb_dict['app_name'] }}
+TAG = latest
+USER=$(shell id -u)
+PWS=$(shell pwd)
+build_and_push: apb_build docker_push apb_push
+
+.PHONY: apb_build
+apb_build:
+	docker run --rm --privileged -v $PWD:/mnt -v $HOME/.kube:/.kube -v /var/run/docker.sock:/var/run/docker.sock -u $UID docker.io/ansibleplaybookbundle/apb:latest prepare
```

Comment:
hrm ... I add to change that line to:

```
docker run --rm --privileged -v $(PWD):/mnt:z -v $(HOME)/.kube:/.kube -v /var/run/docker.sock:/var/run/docker.sock -u $(USER) docker.io/ansibleplaybookbundle/apb:latest prepare
```

locally in my _own_`Makefile`

Reply:
**NOTE:** I've added the `:z` suffix to the volume mount.

From `man docker-run` on Fedora26:

>  Labeling systems like SELinux require that proper labels are placed on volume content mounted into a container. Without a label, the security system might prevent the processes running inside the
       container from using the content. By default, Docker does not change the labels set by the OS.

       To change a label in the container context, you can add either of two suffixes :z or :Z to the volume mount. These suffixes tell Docker to relabel file objects on the shared volumes. The z option
       tells Docker that two containers share the volume content. As a result, Docker labels the content with a shared content label. Shared volume labels allow all containers to read/write content.  The Z
       option tells Docker to label the content with a private unshared label.  Only the current container can use a private volume.